sqlglot/parser.py

```diff
@@ -1104,21 +1110,13 @@ def _parse_create(self) -> t.Optional[exp.Expression]:
             this=this,
             kind=create_token.text,
             expression=expression,
-            set=set_,
-            multiset=multiset,
-            global_temporary=global_temporary,
-            volatile=volatile,
             exists=exists,
             properties=properties,
-            temporary=temporary,
-            transient=transient,
-            external=external,
             replace=replace,
-            unique=unique,
-            materialized=materialized,
             data=data,
             statistics=statistics,
             no_primary_index=no_primary_index,
+            preserve_rows=preserve_rows,
```

Comment:
why isn't preserve rows also in properties?

Reply:
Update moves all 4 of those into a new properties location